### ISSUE TITLE
Zen Mode is broken on NodeBB v1.11.1 (?)

### ISSUE THREAD

**Issue by jtsimoes** [NodeBB/nodebb-plugin-composer-default#117]
When I click on 'Zen Mode' seems that nothing happens in the latest version of NodeBB.
Can you check this?

**Comment by julianlam** [comment]
Hi there, I can confirm that zen mode is working fine for me at the moment. Do you see any errors on the client-side (dev console) when you try to invoke zen mode?

**Comment by jtsimoes** [comment]
Yup @julianlam 
On file `screenfull.js` at line 93:
```
 Uncaught (in promise) TypeError: Failed to execute 'requestFullscreen' on 'Element': parameter 1 ('options') is not an object.
    at Object.request (screenfull.js:93)
    at Object.toggle (screenfull.js:103)
    at x.fn.init.zen (formatting.js?v=blhabefcvh6:1)
    at HTMLLIElement.<anonymous> (formatting.js?v=blhabefcvh6:1)
    at HTMLDivElement.dispatch (jquery.js:5183)
    at HTMLDivElement.m.handle (jquery.js:4991)
```

**Comment by julianlam** [comment]
sindresorhus/screenfull.js#111

Fixed in v6.1.21

**Comment by jtsimoes** [comment]
Oh, right... Now I get it.
Thanks!
